examples/tracing/langchain_auto.py

```diff
@@ -0,0 +1,50 @@
+"""
+This example demonstrates how to enable automatic tracing for LangChain.
+
+Note: this example requires the `langchain` and `langchain-openai` package to be installed.
+"""
+
+import json
+import os
+
+from langchain.prompts import PromptTemplate
+from langchain.schema.output_parser import StrOutputParser
+from langchain_openai import OpenAI
```

Comment:
Can we replace `langchain_openai` with `langchain_huggingface` to make this example runnable?

Example

```python
from langchain_huggingface import HuggingFacePipeline

llm = HuggingFacePipeline.from_model_id(
    model_id="gpt2",
    task="text-generation",
    pipeline_kwargs={"max_new_tokens": 50},
)

print(llm.invoke("Hello, my name is"))
```

Reply:
Hmm we are using OpenAI models elsewhere in langchain examples so it's more straight forward IMO.